prog/postgres/postgres_server_nexus.rb

```diff
@@ -291,7 +291,6 @@ def before_run
 
     # create a new set of firewall rules
     postgres_server.create_resource_firewall_rules
-    vm.incr_update_firewall_rules
```

Comment:
Isn't it already unnecessary? We might remove with a separate commit.

Reply:
Yes, probably better to clean it in a separate PR after the deployment to not cause entity inconsistency during deployment. I will create the PR.

Reply:
I mean, when `create_resource_firewall_rules` is called, wasn't it call `vm.incr_update_firewall_rules` itself?

Reply:
No, `create_resource_firewall_rules` calls `Firewall.insert_firewall_rule` which calls `PrivateSubnet.incr_update_firewall_rules` which triggers firewall rule update for all the VMs in the subnet via `vm.incr_update_firewall_rules`. So at this point I'm not sure what we should clean-up.